src/coreComponents/functions/CMakeLists.txt

```diff
@@ -27,7 +27,7 @@ if( ENABLE_MATHPRESSO )
 endif()
 
 
-set( dependencyList ${parallelDeps} codingUtilities dataRepository )
+set( dependencyList ${parallelDeps} codingUtilities dataRepository fileIO )
```

Comment:
I wish we didn't need to add additional dependencies but I guess this one should not be a big issue.